sdk/src/target/local_device.cpp

```diff
@@ -766,3 +766,74 @@ aditof::Status LocalDevice::applyCalibrationToFrame(uint16_t *frame,
 
     return aditof::Status::OK;
 }
+
+aditof::Status LocalDevice::getDeviceFileDescriptor(int &fileDescriptor) {
+    using namespace aditof;
+
+    if (m_implData->fd != -1) {
+        fileDescriptor = m_implData->fd;
+        return Status::OK;
+    }
+
+    return Status::INVALID_ARGUMENT;
+}
+
+aditof::Status LocalDevice::queueBuffer(unsigned int index) {
+    using namespace aditof;
+
+    struct v4l2_buffer buf;
```

Comment:
Shouldn't the buf be cleared (set all bits to 0)?

Reply:
It should. I've added the clear now. Thanks